nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -729,23 +901,30 @@ NiFi properties have resolution precedence of which you should be aware when cre
 * Processor-specific attributes
 * FlowFile properties 
 * FlowFile attributes 
-* From variable registry: 
+* From Variable Registry: 
 ** User defined properties (custom properties)
 ** System properties 
 ** Operating System environment variables
 
 When you are creating custom properties, ensure that each custom property contains a distinct property value,
 so that it is not overridden by existing environment properties, system properties, or FlowFile attributes.
 
-There are two ways to use and manage custom properties:
+There are two ways to use and manage custom properties with Expression Language:
+
+- *Variables*: Variables are created and configured within the NiFi UI. They can be used in any field that supports Expression Language. Variables cannot be used for sensitive properties. NiFi automatically picks up new or modified variables. Variables are defined at the Process Group level, as a result, the access policies for viewing and changing variables are derived from the access policies of the Process Group. See <<Variables>> for more information.
```

Comment:
I made sure to state "two ways to use and manage custom properties **with Expression Language**", which is a category I don't believe Parameters fall under.

But, since Customer Properties File method uses Variables, I changed the note to include that his functionality will be removed also.

Reply:
Thanks for the clarification.